packages/frontend/src/components/versus_match_card.tsx

```diff
@@ -0,0 +1,30 @@
+export function VersusMatchCard(props: { score: Record<string, number> }) {
+  const [firstKey, secondKey] = Object.keys(props.score);
+  const firstValue = props.score[firstKey];
+  const secondValue = props.score[secondKey];
+  return (
+    <div className="bg-gray-400 text-center w-min rounded-lg text-white overflow-hidden">
+      <div
+        className={`${
+          secondValue <= firstValue ? "bg-p-green-200" : ""
```

Comment:
As you can see with the coverage (and in the files changed bit) this branch is not covered, looking at your tests, you never check if the second person is the winner and not the first